src/vues/nouvellesFonctionnalites.pug

```diff
@@ -7,13 +7,47 @@ block append styles
 block main
   .marges-fixes
     .documentation
-      h1 Nouvelles Fonctionnalités
+      h1 Nouvelles fonctionnalités
       p.description Découvrez les dernières évolutions de MonServiceSécurisé.
 
       section.informations.
         Toutes les données déjà renseignées dans vos dossiers en cours et finalisés sont toujours
         conservées et accessibles quelles que soient les améliorations apportées dans l'interface.
 
+      h2 16 octobre 2022
+      ul.evolutions
+        li.
+          #[h4 Une interface repensée pour mieux vous accompagner]
+          Une rubrique « décrire » : toutes les informations indispensables sur votre service. #[br]
+          Une rubrique « sécuriser » : la liste personnalisée des mesures de sécurité à mettre en œuvre, proposée par l'ANSSI pour votre service. #[br]
+          Une rubrique « homologuer » : [ bientôt disponible ] pour générer rapidement une décision d'homologation (PDF) et renseigner la date d'homologation. #[br]
+          Deux outils complémentaires : « risques de sécurité » pour cartographier les impacts potentiels des risques les plus courants ; « contacts utiles ».
+        li.
+          #[h4 L'arrivée de l'indice cyber !]
+          En un coup d'œil, obtenez une évaluation indicative du niveau de sécurité des services numériques référencés
+          sur MonServiceSécurisé, pour l'équipe et l'autorité d'homologation. L'indice cyber est obtenu à partir des
+          mesures mises en œuvre, sur la base des déclarations de l'équipe, par rapport à l'ensemble des mesures proposées
+          par l'ANSSI.
+        li.
+          #[h4 Un nouveau PDF de « synthèse de la sécurité » d'un service]
+          À la demande générale, une nouvelle version du PDF faisant la « synthèse de la sécurité » permettra de faire
+          le point sur les mesures de sécurité mises en œuvre et d'afficher l'indice cyber. Les annexes détaillées
+          (ex : liste des mesures de sécurité, risques) seront déployées dans les semaines à venir.
+        li.astuce Vous pouvez continuer à télécharger le PDF de synthèse et d'homologation dans sa première version depuis la synthèse de vos services.
+        li #[h4 L'inscription 100% en ligne et autonome pour tous les agents publics]
+        li.
+          #[h4 Le référencement possible de tous les sites web, applications mobiles et API publics]
+          Pilotez la sécurité de tous vos services publics numériques depuis votre espace personnel dans MonServiceSécurisé.
+          Seuls les services numériques qui relèvent d'une réglementation encadrant la sécurité des systèmes d'information
+          aux besoins de sécurité élevés ne peuvent pas être référencés sur MonServiceSécurisé.
+          #[a(href = '/questionsFrequentes', target = '_blank', rel = 'noopener') En savoir plus]
+        li
+          .
+            #[h4 La rubrique « Profil » devient « Mes informations »]
```

Comment:
Ici tu peux utiliser

```pugjs
li
  h4 titre
  .
    texte
  img
```

Reply:
Merci. J'ai changé sur tous les `li` par souci d'homogénéité. Je trouve que le `pug` devient plus compréhensible comme ça.